docs/development/docs/code-style.md

```diff
@@ -0,0 +1,128 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Backend
```

Comment:
```suggestion
## Backend
```

In `mkdocs`, the h1-header is used as the page title. Therefore, it should not be used (unless you want to have another page title). All titles should use the next level (h1 -> h2, h2->h3, ...).